internal/cleanup/service_logs.go

```diff
@@ -8,15 +8,15 @@ import (
 	"github.com/pkg/errors"
 
 	"github.com/elastic/elastic-package/internal/files"
-	"github.com/elastic/elastic-package/internal/install"
+	"github.com/elastic/elastic-package/internal/locations"
```

Comment:
I'm not sure if `internal/locations` is the correct place for this feature. This is a configuration that's strongly related to app configuration. I would be for placing it all together under `internal/configuration`, so we may have:
`internal/configuration/locations`
`internal/configuration/profiles`
`internal/configuration/stack`

In this case the goal would be transforming the `internal/install` into something more sophisticated that can deal with profiles and config extensions.